Makefile

```diff
@@ -88,6 +88,10 @@ GAIA_DEV_PIXELS_PER_PX?=1
 NGA_SERVICE_WORKERS?=0
 FIREFOX_SYNC?=0
 
+ifeq ($(GAIA_DEVICE_TYPE),tv)
+  FIREFOX_SYNC=1
+endif
+
```

Comment:
I'd prefer to add a `tv.mk` (or such) to `build/config/tv` and include it from the global `Makefile`